receiver/awscontainerinsightreceiver/internal/stores/podstore_test.go

```diff
@@ -291,6 +292,90 @@ func TestPodStore_addContainerCount(t *testing.T) {
 	assert.Equal(t, int(1), metric.GetField(ci.MetricName(ci.TypePod, ci.ContainerCount)).(int))
 }
 
+const (
+	PodFailedMetricName      = "pod_status_failed"
```

Comment:
Its going to be hard for customers to know whats a phase and whats a condition. I'd recommend including it in the name..
so `pod_status_phase_*` and `pod_status_condition_*`.

Reply:
??? is that a new decision made? initial requirements asked for these namings

Reply:
These recommendations were in the doc in the appendix before this task was picked up. And the task references the doc that has a comment referencing this on the same lines as the metrics. 

Either way, this can be fixed after if you think its a big change